1.9/administration/upgrading.md

```diff
@@ -11,6 +11,7 @@ This document provides instructions for upgrading a DC/OS cluster from version 1
 
 - The Advanced Installation method is the _only_ recommended upgrade path for DC/OS. It is recommended that you familiarize yourself with the [Advanced DC/OS Installation Guide][advanced-install] before proceeding.
 - The [VIP features](/docs/1.9/usage/service-discovery/load-balancing-vips/virtual-ip-addresses/), added in DC/OS 1.8, require that ports 32768 - 65535 are open between all agent and master nodes for both TCP and UDP.
+- Overlay networks require Docker 1.11. For more information, see the [documentation](/docs/1.9/administration/overlay-networks/).
```

Comment:
Are overlay networks new? Or do you mean if you're upgrading with overlay networks? or both? More generally, what is the context?